extensions/ql-vscode/src/pure/helpers-pure.ts

```diff
@@ -38,6 +38,12 @@ export const asyncFilter = async function <T>(arr: T[], predicate: (arg0: T) =>
  */
 export const REPO_REGEX = /^(?:[a-zA-Z0-9]+-)*[a-zA-Z0-9]+\/[a-zA-Z0-9-_]+$/;
 
+/**
+ * This regex matches GiHub organization and user strings. These are made up for alphanumeric 
+ * characters or single hyphens, starting and ending in an alphanumeric character.
+ */
+export const OWNER_REGEX = /^(?:[a-zA-Z0-9]+-)*[a-zA-Z0-9]+$/;
```

Comment:
It might be worth making this regex slightly more permissive, to account for older names that follow a different pattern. See https://github.com/github/vscode-codeql/pull/1391#discussion_r899271336, where this came up yesterday!

Reply:
Ah thanks, updated!